backend/public-graph/graph/resolver.go

```diff
@@ -3434,3 +3444,51 @@ func (r *Resolver) isExcludedError(ctx context.Context, errorFilters []string, e
 	}
 	return false
 }
+
+func (r *Resolver) Login(w http.ResponseWriter, req *http.Request) {
+	var credentials LoginCredentials
+
+	if ADMIN_PASSWORD == "" {
+		http.Error(w, "", http.StatusInternalServerError)
+		return
+	}
+
+	ctx := req.Context()
+
+	err := json.NewDecoder(req.Body).Decode(&credentials)
+
+	if err != nil {
+		log.WithContext(ctx).Error(err)
+		http.Error(w, "invalid credentials", http.StatusBadRequest)
+		return
+	}
+
+	if ADMIN_PASSWORD != credentials.Password {
+		http.Error(w, "invalid credentials", http.StatusBadRequest)
+		return
+	}
+
+	atClaims := jwt.MapClaims{}
+	atClaims["authorized"] = true
+	atClaims["claim"] = time.Now().Add(time.Hour * 24).Unix()
```

Comment:
can this be a constant near the `ADMIN_PASSWORD` that is describing that this is the password auth expiry?

Reply:
Done